tests/NHSD.BuyingCatalogue.Ordering.Api.Testing.Data/Entities/OrderItemEntity.cs

```diff
@@ -81,5 +81,17 @@ FROM dbo.OrderItem
 
             return await SqlRunner.QueryFirstAsync<OrderItemEntity>(connectionString, sql, new { orderId, catalogueItemId });
         }
+
+        public static async Task<int> GetItemIdsForOrderId(
```

Comment:
I think naming this method `GetOrderItemCountForOrder` or something similar might be a bit clearer.

Reply:
done